conf/local.conf.append

```diff
@@ -0,0 +1,3 @@
+# Comment to disable and Uncomment to enable iot feature
+USER_FEATURES += "java"
```

Comment:
shouldn't this be IMAGE_FEATUR?

Reply:
I think it should be IMAGE_FEATURES as it will end up from there in DISTRO_FEATURES and we have used same approach to provide iot feature control to users

Reply:
@abelal has proposed that we converge on `USER_FEATURES` as a consistent way to enable optional product features regardless of what variables control it under the hood (`DISTRO_FEATURES`, `EXTRA_IMAGE_FEATURES`, `MACHINE_FEATURES`, etc). I not only like this idea for this case, I rather like the idea of converging on it for features which are currently enabled via `-l` with setup-environment as well. We'll have to make sure that we continue to use the correct underlying variables to control the behaviors, leaving only `USER_FEATURES` as a user interface to it, to ensure we still align with Yocto Project conventions in our implementations, though. @Noor-Ahsan , thoughts? For what it's worth, the use of `USER_FEATURES` in this PR has a +1 from me.

Reply:
I am fine with it.